### ISSUE TITLE
Store Photoswipe object in component's data object

### ISSUE THREAD

**Issue by pavloniym** [rap2hpoutre/vue-picture-swipe#18]
Hello, thank you for your package!
What about storing photoswipe object in component's data object?
This will give access to it's API,events and methods from $ref.

**Comment by rap2hpoutre** [comment]
Hi and thank you for your contribution (and sorry for *very late* answer). That's a good idea. I keep this issue open and will try to tackle this soon. Still, feel free to submit a PR if you want!

**Comment by oller** [comment]
I believe this issue would solve the issue i'm facing.

I'm trying to set the following function in the options prop: (lifted from photoswipe docs)

```js
getImageURLForShare: function( shareButtonData ) {
    // `shareButtonData` - object from shareButtons array
    // 
    // `pswp` is the gallery instance object,
    // you should define it by yourself
    // 
    return pswp.currItem.src || '';
},
```

For this to work, I need to access the gallery instance object (`pswp`), which I was hoping would be exposed perhaps via a $ref, but I don't believe it's possible with the current implementation?

**Comment by oller** [comment]
Have created a PR to make this possible, only 2 lines:

https://github.com/rap2hpoutre/vue-picture-swipe/pull/22

**Comment by rap2hpoutre** [comment]
Fixed thanks to @oller. Available in 1.2.0

**Comment by oller** [comment]
Great work, thanks for the quick merge and release! @rap2hpoutre 👍
